src/script/interpreter.cpp

```diff
@@ -1580,7 +1581,7 @@ bool EvalScript(vector<vector<unsigned char> >& stack, const CScript& script, un
                                     return set_error(serror, SCRIPT_ERR_WITHDRAW_VERIFY_OUTPUT_SCRIPT);
 
 #ifndef BITCOIN_SCRIPT_NO_CALLRPC
-                                if (GetBoolArg("-validatepegin", DEFAULT_VALIDATE_PEGIN) && !checker.IsConfirmedBitcoinBlock(genesishash, merkleBlock.header.GetHash(), flags & SCRIPT_VERIFY_INCREASE_CONFIRMATIONS_REQUIRED))
+                                if (GetBoolArg("-validatepegin", DEFAULT_VALIDATE_PEGIN) && !checker.IsConfirmedBitcoinBlock(genesishash, merkleBlock.header.GetHash(), flags & SCRIPT_VERIFY_INCREASE_CONFIRMATIONS_REQUIRED, Params().GetConsensus().nMinDepth))
```

Comment:
Why does this not just use `GetArg("-parentconfirmationdepth",..)` instead of including chainparams? The two should be identical always, right?

Reply:
In that case, you want to also use the consensus value in places where you reference this. Right now you are sometimes using `GetArg` and sometimes `GetConsensus()`. Won't that result in inconsistencies?

Reply:
I don't see additional instances, other than init.cpp. Could you point them out?

Reply:
Oops, you're right.